Physics/Assets/Scripts/Camera/CustomCamera.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System.Collections;
+using System.IO;
+using UnityEngine;
+
+public class CustomCamera : MonoBehaviour
+{
+    /// <summary>
+    /// Camera component connected to the current GameObject.
+    /// Used for rendering.
+    /// </summary>
+    private Camera _camera;
+
+    /// <summary>
+    /// Renderer coroutine used to take continuous screenshots at intervals.
+    /// Used as a parameter for StopCoroutine.
+    /// </summary>
+    private Coroutine _rendererCoroutine;
+
+    private void OnEnable()
+    {
+        SaveCamera();
+        if (_camera == null)
+        {
+            GetComponent<CustomCamera>().enabled = false;
```

Comment:
You can leave out the call to GetComponent, and just have enabled=false.

Reply:
Done.